packages/arb-util/broadcastclient/broadcastclient.go

```diff
@@ -101,15 +101,19 @@ func (bc *BroadcastClient) backgroundReader(messageReceiver chan broadcaster.Bro
 			return
 		}
 
-		logger.Debug().Int("length", len(msg)).Msg("received broadcast message")
-
 		res := broadcaster.BroadcastMessage{}
 		err = json.Unmarshal(msg, &res)
 		if err != nil {
 			logger.Error().Err(err).Msg("error unmarshalling message")
 			continue
 		}
 
+		if len(res.Messages) > 0 {
+			logger.Debug().Hex("acc", res.Messages[0].FeedItem.BatchItem.Accumulator.Bytes()).Msg("received batch item")
```

Comment:
I'm not sure if we ever send more than one message at a time, but we should probably log them or at least how many there are.

Reply:
sounds good